src/main/java/org/numenta/nupic/SDR.java

```diff
@@ -85,9 +85,7 @@
      * @return  the column indexes of the specified cells.
      */
     public static int[] cellsToColumns(List<Cell> cells, int cellsPerColumn) {
-        IntStream op = cells.stream().mapToInt(c -> c.getIndex());
-            
-        return op.map(cellIdx -> cellIdx / cellsPerColumn).distinct().toArray();
+        return cells.stream().mapToInt(c -> c.getColumn().getIndex()).distinct().toArray();
     }
```

Comment:
Well, I'd remove it but that'd be a breaking change to the API (same applies to your next comment). These functions are used all over the client code. The compiler will optimise that away, so no real need to change it until you decide to change the API.